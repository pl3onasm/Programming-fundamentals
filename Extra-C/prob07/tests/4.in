10
01??0?0??1
