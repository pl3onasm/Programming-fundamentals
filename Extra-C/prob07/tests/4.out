0101010011
0101010101
0110010011
0110010101
